tests/kafkatest/automq/quota_test.py

```diff
@@ -0,0 +1,224 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from ducktape.mark.resource import cluster
+from ducktape.tests.test import Test
+from ducktape.mark import matrix
+
+from kafkatest.services.console_consumer import ConsoleConsumer
+from kafkatest.services.kafka import KafkaService
+from kafkatest.services.performance import ProducerPerformanceService
+from kafkatest.version import DEV_BRANCH
+from kafkatest.automq.automq_e2e_util import publish_broker_configuration
+
+
+class QuotaTest(Test):
+    """
+    These tests verify that Broker quota provides expected functionality.
+    """
+    RECORD_SIZE = 3000
+
+    def __init__(self, test_context):
+        super(QuotaTest, self).__init__(test_context=test_context)
+
+        self.topic = 'test_topic'
+        self.producer_client_id = 'producer_client'
+        self.consumer_client_id = 'consumer_client'
+
+        self.client_version = DEV_BRANCH
+
+        self.broker_id = '1'
+
+        self.create_kafka(test_context)
+
+        self.maximum_client_deviation_percentage = 100.0
+        self.maximum_broker_deviation_percentage = 50
+
+        self.success = True
+        self.msg = ''
+
+
+    def create_kafka(self, test_context):
+        log_size = 256 * 1024 * 1024
+        block_size = 256 * 1024 * 1024
+        server_prop_overrides = [
+            ['broker.quota.enabled', 'true'],
```

Comment:
Add and verify the static configuration.

- broker.quota.produce.bytes
- broker.quota.fetch.bytes